distro/runtime/concierge/smarthome.xargs

```diff
@@ -23,6 +23,7 @@
 # -Dorg.eclipse.concierge.debug.packages=true
 # -Dorg.eclipse.concierge.debug.services=true
 # -Dorg.eclipse.concierge.debug.classloading=true
+-Dorg.eclipse.concierge.debug.resolver=true
```

Comment:
I would comment it out: it was just missing as an option which could be activated
By default it should be NOT active
So: 
`# -Dorg.eclipse.concierge.debug.resolver=true`